lib/config/migration.ts

```diff
@@ -19,7 +20,9 @@ import { mergeChildConfig } from './utils';
 const options = getOptions();
 
 let optionTypes: Record<string, RenovateOptions['type']>;
-
+const afterBeforeRe = regEx(
+  /^(.*?)(after|before) (.*?) and (after|before) (.*?)( |$)(.*)/
+);
```

Comment:
@viceice do you think we need to push these regEx up to module scope or can they be located more close to where they're used?

Reply:
@viceice Could you please review this once?

Reply:
I think this can now be moved more close to usage now